client/package.json

```diff
@@ -4,8 +4,8 @@
   "author": "Nadav Ivgi",
   "license": "MIT",
   "scripts": {
-    "postinstall": "../node_modules/.bin/patch-package",
-    "dist": "../node_modules/.bin/babel -d dist src"
+    "postinstall": "cd ../node_modules/.bin &&  patch-package",
+    "dist": "cd ../ && cd node_modules && cd .bin || babel -d dist src"
```

Comment:
What is this `||` (OR) condition based on? Did you mean `&&`?

Reply:
Right I should be &&.
I'll fix it